### DISCUSSION TITLE
GitHub Desktop and Git GUI desktop

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36724] [Other Feature Feedback, Questions, & Ideas] [answered]
It seems like the two apps are doing the same thing on my laptop, and Git GUI is not friendly to use. Any chance I can just keep the GitHub desktop app?

Any comments are welcomed, and other related tools and topic recommendation are all opened to.

**Comment by ossanna16** [comment]
Hey there @Robertblazor and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by ghost** [comment, marked as answer]
No, I tried, the GitHub pull request extension stopped working on my VSCode.
